taipy/gui/viselements.json

```diff
@@ -877,6 +877,35 @@
                 ]
             }
         ],
+        [
+            "progress",
+            {
+                "properties": [
+                    {
+                        "name": "linear",
+                        "type": "bool",
+                        "doc": "If True, then a linear element is rendered instead of a circular one."
```

Comment:
Should I mark the `default_value` as `True` or `False`? Because in the [above comment](https://github.com/Avaiga/taipy/pull/1303#discussion_r1640382907), `True` has been mentioned as default value?

Reply:
The default value being False, we want to focus on documenting the False case.
My proposal works then.